tests/run-make/raw-dylib-inline-cross-dylib/Makefile

```diff
@@ -4,15 +4,24 @@
 
 include ../../run-make-fulldeps/tools.mk
 
+# We're using the llvm-nm instead of the system nm to ensure it is compatible
+# with the LLVM bitcode generated by rustc.
+# Except on Windows where piping/IO redirection under MSYS2 is wonky with llvm-nm.
+ifndef IS_WINDOWS
+NM = "$(LLVM_BIN_DIR)"/llvm-nm
+else
+NM = nm
+endif
+
 all:
 	$(RUSTC) --crate-type dylib --crate-name raw_dylib_test lib.rs -C prefer-dynamic
 	$(RUSTC) --crate-type dylib --crate-name raw_dylib_test_wrapper lib_wrapper.rs -C prefer-dynamic
 	$(RUSTC) --crate-type bin driver.rs -L "$(TMPDIR)" -C prefer-dynamic
 	# Make sure we don't find an import to the functions we expect to be inlined.
-	"$(LLVM_BIN_DIR)"/llvm-objdump -p $(TMPDIR)/driver.exe | $(CGREP) -v -e "inline_library_function"
-	"$(LLVM_BIN_DIR)"/llvm-objdump -p $(TMPDIR)/driver.exe | $(CGREP) -v -e "inline_library_function_calls_inline"
+	$(NM) -p $(TMPDIR)/driver.exe | $(CGREP) -v -e "inline_library_function"
+	$(NM) -p $(TMPDIR)/driver.exe | $(CGREP) -v -e "inline_library_function_calls_inline"
 	# Make sure we do find an import to the functions we expect to be imported.
-	"$(LLVM_BIN_DIR)"/llvm-objdump -p $(TMPDIR)/driver.exe | $(CGREP) -e "library_function"
+	$(NM) -p $(TMPDIR)/driver.exe | $(CGREP) -e "library_function"
```

Comment:
Why did you switch to `nm` instead of `objdump`?

Reply:
I was in hurry and copied the solution form the other test. Should be fine now.